tensorflow/python/keras/losses.py

```diff
@@ -1688,6 +1689,41 @@ def sparse_categorical_crossentropy(y_true, y_pred, from_logits=False, axis=-1):
       y_true, y_pred, from_logits=from_logits, axis=axis)
 
 
+@dispatch.dispatch_for_types(sparse_categorical_crossentropy,
+                             ragged_tensor.RaggedTensor)
+def _ragged_tensor_sparse_categorical_crossentropy(
+    y_true, y_pred, from_logits=False, axis=-1):
+  """ Implements support for handling RaggedTensors.
+
+      Expected y_pred shape: (batch, sequence_len, n_classes) with sequence_len
+      being variable per batch.
+      Return shape: (batch, sequence_len).
+
+      When used by SparseCategoricalCrossentropy() with the default reduction
+      (SUM_OVER_BATCH_SIZE), the reduction averages the loss over the
+      number of elements independent of the batch. E.g. if the RaggedTensor
+      has 2 batches with [2, 1] values respectivly the resulting loss is
```

Comment:
There is a typo - respectiv**e**ly; and I believe there should be a comma after respectively.

Reply:
Done.